grails-app/controllers/org/pih/warehouse/user/UserController.groovy

```diff
@@ -118,24 +118,15 @@ class UserController {
      */
     def save() {
         log.info "attempt to save the user; show form with validation errors on failure"
-        def userInstance = new User(params)
-
-        // Default value for active field on Person is set to True
-        // which is inherited by User
-        // but when creating a new user default value should be set to False
-        if (!params.hasProperty("active")) {
-            userInstance.active = false
+        User userInstance = new User(params)
+        User persistedUser = userService.addUser(userInstance, params)
```

Comment:
Following our convention we should use save<DomainName> for the service.
```
saveUser(User user)
```
If there's a need to separate create and update then we can add two methods, but in this case we don't need that just yet.